### DISCUSSION TITLE
secret function I need help

### DISCUSSION THREAD

**Discussion by Larswagner1987** [danielnilsson9/bbs-fw#193] [Ideas]
@danielnilsson9 @ how do I get it set when I turn on the light button so that I can call up my settings and when I turn off the light that the traffic jam is there again??? I don't understand . Can someone tell me how to set this up???? Thank you .

**Comment by WvBijsterveld** [comment]
Man honestly we’re gonna need u to slowly explain and reword this for anyone to understand, will be happy to help then.

**Reply by Larswagner1987** [reply]
I haven't read it but unfortunately you can't find much, it would be good to have a short description of how to set it up, I only have a display that shows the km/h and can turn the levels on and off like lit and that's all. Display is a Bafang DZ41 LCD display.

**Reply by Larswagner1987** [reply]
Light on my Profiel, light off stvo setting back there, how can you set that.

**Reply by WvBijsterveld** [reply]
i think you mean the sport mode toggle.
Is that correct?
(switching between 2 sets of assist levels, i.e. one is 25km/h restricted and one is 100km/h "restricted")

**Reply by Larswagner1987** [reply]
with the wattage I meant, so when I turn on the light, then my 1150Watt are there and when it is off, then my 250Watt are in it, can you also set it that way ????

**Reply by WvBijsterveld** [reply]
if yes: after flashing BBSFW to your controller you can open the included Tool for it. connect your controller, go to Assist levels tab,
bottom right you'll see operation mode toggle then set this to lights toggle you'll get that.
then configure your standard and sport levels as you like and it's done.
There's not really much we can do for your display but every input/output should be the same(turning lights on/off,  walk mode, using assist levels, etc.)

**Reply by WvBijsterveld** [reply]
however if you use lights toggle you would not be able to use your actual light on your bike(given it is connected to your motor) if you would like to use your light you can set the lights toggle + a PAS level, if you are in set PAS level you will change your 1150W-250W, if in any other you will change your light like normal.

**Reply by Larswagner1987** [reply]
[so I have to Sport Profiel in the program my values übertragren and at Standard Profiel the Stvo values enter, and then set below Operation Mode Toggle light button Right ?and Startup Assist Level on zero leave.when I turn on the light on the display should then be the Protprifiel there:have I understood correctly ?](https://www.google.com/search?client=firefox-b-d&sca_esv=569828632&sxsrf=AM9HkKnuoVNBvGNbdpEzll-tJqhrpcwwkw:1696150426819&q=so+I+have+to+Sport+Profiel+in+the+program+my+values+%C3%BCbertragren+and+at+Standard+Profiel+the+Stvo+values+enter,+and+then+set+below+Operation+Mode+Toggle+light+button+Right+?and+Startup+Assist+Level+on+zero+leave.when+I+turn+on+the+light+on+the+display+should+then+be+the+Protprifiel+there:have+I+understood+correctly+?&spell=1&sa=X&ved=2ahUKEwjJ6rPdvNSBAxUVgP0HHcd8CgYQBSgAegQICRAB)

**Reply by WvBijsterveld** [reply]
yes

**Reply by Larswagner1987** [reply]
Thanks for helping

**Comment by WvBijsterveld** [comment]
no problem!
